tutorials/tour/basics.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: tutorial
+title: Basics
+
+disqus: true
+
+tutorial: scala-tour
+num: 2
+next-page: unified-types
+previous-page: tour-of-scala
+---
+
+In this page, we will cover basics of Scala.
+
+## Variables
+
+You can define variables with `var` keyword.
+
+```
+var x = 1 + 1
+x += 1
+```
+
+Often, you want your variables to be immutable. You can use `val` keyword in that case.
+
+```
+val x = 1 + 1
+x += 1 // This does not compile because you declared the variable with "val" keyword
+```
+
+Type of variables can be inferred, but you can also explicitly state type like below.
+
+```
+val x: Int = 1 + 1
+```
+
+## Functions
+
+You can define functions like below.
+
+```
+(name: String) => println("Hello, " + name + "!")
```

Comment:
This example introduces a few things that might not be obvious coming from certain languages:

- Anonymous functions
- Type declaration after identifier
- Return type inference 
- Lack of "return" keyword

I think it's important to explain all of those or at least declare the return type and not make the function anonymous. Otherwise, it could be a bit cryptic.